newtab.css

```diff
@@ -0,0 +1,4 @@
+body {
+    background-image: url("https://source.unsplash.com/random/3840x2160");
+    background-size: cover;
+} 
```

Comment:
Instead of just using the body, you could add in two elements here - one for the background like you have it, and another for an overlay. An overlay would allow you to darken the image and then you could put bold white text over top and it would be easy to read.

Here's one way to do that:
```html
<main>
   <img src="https://bit.ly/2rlzaXi">
   <section>
      <h1>A title here</h1>
      <h2>Some content, maybe the time</h2>
      <button>A link?</button>
   </section>
</main>
```

```css
main {
    height: 100vh;
    width: 100vw;
    background: black;
    overflow: hidden;
}

img {
   object-fit: cover;
   opacity: 0.4;
}
```

Reply:
Understood!!